models/open_llama/dataloop.json

```diff
@@ -0,0 +1,103 @@
+{
+    "name": "openllama-huggingface-app",
+    "description": "openllama for chatting - HF",
+    "attributes": {
+        "provider": "Hugging Face",
+        "license": "apache-2.0",
+        "category": "Model",
+        "Computer Vision": "NLP"
+    },
+    "displayName": "Openllama Huggingface App",
+    "version": "v0.0.1",
+    "scope": "project",
+    "codebaseZ": {
+        "type": "git",
+        "gitUrl": "https://github.com/dataloop-ai-apps/nvidia-models.git",
+        "gitTag": "v0.0.1"
+    },
+    "codebase": {
+        "type": "local",
+        "src_path": "../.."
+    },
+    "components": {
+        "computeConfigs": [
+            {
+                "name": "openllama-huggingface-deploy",
+                "versions": {
+                    "dtlpy": "1.89.6"
+                },
+                "secrets": [],
+                "runtime": {
+                    "podType": "gpu-t4",
+                    "concurrency": 1,
+                    "runnerImageX": "gcr.io/viewo-g/piper/agent/runner/gpu/nvidia-tao:0.1.2",
```

Comment:
wrong docker image

Reply:
runnerImageX means not used, but you right will remove, no docker is needed